crates/air/tests/integration/format.rs

```diff
@@ -0,0 +1,205 @@
+use std::process::Command;
+
+use tempfile::TempDir;
+
+use crate::helpers::binary_path;
+use crate::helpers::path_root;
+use crate::helpers::relative_path_fixtures;
+use crate::helpers::CommandExt;
+
+#[test]
+fn test_default_options() -> anyhow::Result<()> {
+    let directory = TempDir::new()?;
+    let directory = directory.path();
+
+    let path = "test.R";
+    std::fs::write(directory.join(path), "1 + 1")?;
+
+    let output = Command::new(binary_path())
+        .current_dir(directory)
+        .arg("format")
+        .arg(path)
+        .run();
+
+    assert!(output.status.success());
```

Comment:
Here's where we call the `air` binary. `binary_path()` and `run()` are my own creations, but mostly it's a standard `Command` call, pretty cool!

We can either inspect the `output` manually, or `insta::assert_snapshot!()` it (as i do in `--check` tests below)